src/main/java/klerer/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,34 @@
+package klerer.projectile;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent {
+
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+        g.drawString("(100, 100)", 100, -100);
+        g.setColor(Color.GREEN);
+        g.drawLine(0, 0, getWidth(), -getHeight());
+        g.drawRect(200, -200, 50, 50);
+        g.setColor(Color.MAGENTA);
+        g.fillRect(400, -400, 25, 25);
+        g.setColor(Color.ORANGE);
+        g.drawOval(200, -200, 50, 50);
+
+        g.drawLine(0, 0, (int) projectile.getInterceptX(), -(int) projectile.getPeakY());
```

Comment:
What is this line supposed to do?

Reply:
draw the projectile

Reply:
I don't understand what that means.

Reply:
take the informations from projectile frame and draw a line using that information

Reply:
From where to where?
What are the values?

Reply:
never mind. It makes a line but only goes until the peak. I need to spit it into short lines and fix this. I dont know how to do this.

Reply:
The homework is to figure out how to do it.

Reply:
👍